wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js

```diff
@@ -175,6 +177,24 @@ define(
 					if (~~elData(reactionTypeButton, 'is-assignable') === 0) {
 						elShow(reactionTypeButton);
 					}
+					
+					this._scrollReactionIntoView(reactionTypeButton);
+				}
+			},
+			
+			_scrollReactionIntoView: function (reactionTypeButton) {
+				var scrollableContainer = elBySel('.reactionPopoverContent', this._getPopover());
+				
+				// Do not scroll if the button is located in the upper 75%.
+				if (reactionTypeButton.offsetTop < scrollableContainer.clientHeight * 0.75) {
+					scrollableContainer.scrollTop = 0;
+				}
+				else {
+					// `Element.scrollTop` permits arbitrary values and will always clamp them to
+					// the maximum possible offset value. We can abuse this behavior by calculating
+					// the values to place the selected reaction in the center of the popover,
+					// regardless of the offset being out of range.
+					scrollableContainer.scrollTop = reactionTypeButton.offsetTop + reactionTypeButton.clientHeight / 2 - scrollableContainer.clientHeight / 2;
```

Comment:
```suggestion
					scrollableContainer.scrollTop = reactionTypeButton.offsetTop + (reactionTypeButton.clientHeight / 2) - (scrollableContainer.clientHeight / 2);
```
I prefer parentheses whenever operator with different precedence are being used in a single expression.